app/assets/stylesheets/components/sub_nav.scss

```diff
@@ -10,21 +9,36 @@
     padding-bottom: 10px;
   }
 
+  .nav-links {
+    display: flex;
+    justify-content: space-around;
+
+    @media (min-width: 640px) {
+      justify-content: flex-end;
+    }
+  }
+
   nav {
     text-align: right;
+    margin-top: 4px;
+
+    > :first-child {
+      margin-left: 0px;
+    }
```

Comment:
This ensures that the padding-left that spaces out the subnav links only applies between the links, not on the left of the first link.